src/server.js

```diff
@@ -138,20 +144,30 @@ let load = loader({
       // Create server and start listening
       let server = http.createServer(app);
       await new Promise((accept, reject) => {
-        server.listen(cfg.server.port, accept);
+        server.listen(cfg.server.port);
+        server.once('listening', () => {
+          console.log("Listening on port: " + cfg.server.port);
+        });
         server.once('error', reject);
+        server.once('close', accept);
```

Comment:
I was trying to work things out so that the server would run forever before the promise resolved, while the scanner could run to completion and then resolve its promise.  I'm happy to hear of a better way to accomplish that.